ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -661,7 +661,7 @@ def test_ipa_healthcheck_revocation(self):
         Ensure that healthcheck reports when IPA certs are revoked.
         """
         error_msg = (
-            "Certificate tracked by {key} is revoked {revocation_reason}"
+            "Certificate is revoked, unspecified"
```

Comment:
This line should not be changed upstream. Please see commit https://github.com/freeipa/freeipa-healthcheck/commit/1188d5beaf4a0fdfe98608d9259532b860dabeb1